### ISSUE TITLE
Subfont multiple html files with the same css font's file,  only first  html file work.

### ISSUE THREAD

**Issue by suziwen** [Munter/subfont#83]
```
subfont --no-fallbacks  --output dist a.html b.html
```

a.hmtl file
```
<link href="css/c.css">
```

b.html file

```
<link href="css/c.css">
```

c.css file

```
@font-face....
```

The result only a.html file work, and b.html dont contain any css files.

**Comment by Munter** [comment]
This certainly sounds like a bug. Can you create a PR that puts a reduced example or your html and css in https://github.com/Munter/subfont/tree/master/testdata so I can write a test for it?

**Comment by papandreou** [comment]
Fixed in 4.2.1 🤗

Thanks for reporting.
